src/include/test/unit_vamana_index.cc

```diff
@@ -119,26 +134,47 @@ TEST_CASE("vamana: diskann", "[vamana]") {
   }
 
   auto f = read_diskann_data(diskann_test_data_file);
-  CHECK(f.num_cols() == 256);
-  CHECK(f.num_rows() == 128);
-  auto med = detail::graph::medioid(f);
+  CHECK(num_vectors(f) == 256);
+  CHECK(dimension(f) == 128);
+  auto med = ::medoid(f);
   std::cout << "med " << med << std::endl;
   CHECK(med == 72);
+  // tiledb::Context ctx;
+  // write_matrix(ctx, f, "/tmp/diskann_test_data_file.tdb");
 }
 
 TEST_CASE("vamana: small256 build index", "[vamana]") {
-  auto vindex = detail::graph::vamana_index<float, uint32_t>(256, 50, 0);
+  // DiskANN rust code has this test:
+  // TEST_DATA_FILE = tests/data/siftsmall_learn_256pts.fbin
+  // assert_eq!(visited_nodes.len(), 1);
+  // assert_eq!(scratch.best_candidates.size(), 1);
+  // assert_eq!(scratch.best_candidates[0].id, 72);
+  // assert_eq!(scratch.best_candidates[0].distance, 125678.0_f32);
+  // assert!(scratch.best_candidates[0].visited);
+  // Load 256 points
+  // search_list_size == 50, max degree == 4, alpha == 1.2
+  // medoid == 72, query == 0 ??
+
+  // num_nodes, Lbuild, Rmax_degree
+
+  set_noisy(false);
+
+  auto vindex = vamana_index<float, uint32_t>(256, 50, 4);
   auto x = read_diskann_data(diskann_test_data_file);
   auto graph = read_diskann_mem_index_with_scores(
       diskann_mem_index, diskann_test_data_file);
 
   vindex.train(x);
 
+  set_noisy(false);
```

Comment:
(Minor) I don't think we need `set_noisy(false)`?

Reply:
This is to leave a breadcrumb for toggling on and off.